mvp-spec/model-updater/generate-model-markdown.sh

```diff
@@ -1,16 +1,29 @@
 #sh
 
 BASEDIR=$(dirname "$0")
+TMP_DIR="tmp-schemas"
```

Comment:
maybe add this to `.gitignore`, just in case when the script fails and leaves the directory non empty?

Reply:
Ok